ui/desktop/app/styles/app.scss

```diff
@@ -530,3 +530,110 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     background-color: var(--token-color-palette-neutral-50);
   }
 }
+
+.session-details {
+  .rose-layout-body-content {
+    margin-top: 1.5rem;
+  }
+}
+
+.credential-section-header {
```

Comment:
The markup classes and semantic elements are IMHO a bit confusing. I will start a thread on slack to discuss it 😉

Reply:
Discussed and refactored.